src/sources/ldap.rs

```diff
@@ -90,9 +90,9 @@ impl LdapSource {
 			.any(|flag| status_as_int & flag != 0);
 
 		let ldap_user_id = match read_search_entry(&entry, &self.ldap_config.attributes.user_id)? {
-			// TODO(tlater): Use an encoding that preserves alphabetic order
-			StringOrBytes::Bytes(byte_id) => BASE64_STANDARD.encode(byte_id),
-			StringOrBytes::String(string_id) => BASE64_STANDARD.encode(string_id),
+			// Use hex encoding instead of base64 for consistent alphabetical order
+			StringOrBytes::Bytes(byte_id) => hex::encode(byte_id),
+			StringOrBytes::String(string_id) => hex::encode(string_id.to_lowercase().as_bytes()),
```

Comment:
@emgrav You suggested `base32hex`, was it for a particular reason? I think just a plain `hex` encoding should be sufficient without having to worry about padding.

Reply:
Yes, regular hex also works. base32hex is slightly more space efficient but that's of no concern here.